theme/brb/views/lib/header.hbs

```diff
@@ -1,9 +1,33 @@
+{{#if showAlerts}}
 <header class="mb-1 top-margin">
-	<div class="container">
-		<nav class="navbar-light">
-			<a id="title" class="pull-left" href="/" title="{{$t "global.link.landing"}}">
-				<div class="cloud-logo"></div>
-			</a>
-		</nav>
-	</div>
+    <div class="container">
+        <nav class="navbar-light">
+            <a
+                id="title"
+                class="pull-left"
+                href="/"
+                title="{{$t 'global.link.landing'}}"
+            >
+                <div class="cloud-logo"></div>
+            </a>
+
+            {{> "lib/components/alerts"}}
```

Comment:
maybe just wrap only this line in the {#if showAlerts}}, since there are no other diferences

Reply:
done